tox.ini

```diff
@@ -15,13 +15,17 @@ commands =
     pytest --collect-only
     pytest --color=yes {tty:-s}
 setenv =
+    ANSIBLE_DISPLAY_FAILED_STDERR=1
```

Comment:
Please remove these from the change, they are unrelated, ok for debugging but no reason to make it run spammy.

I think that this change should not touch the tox.ini file.

Reply:
actually, molecule is setting it too. I've put it there to be closer to what's current practice in molecule's ``tox.ini``